gazebo_plugins/include/gazebo_plugins/gazebo_ros_camera_utils.h

```diff
@@ -149,7 +149,8 @@ namespace gazebo
     protected: double cx_prime_;
     protected: double cx_;
     protected: double cy_;
-    protected: double focal_length_;
```

Comment:
This PR is tagetting Melodic, and we can't remove protected members in a released version because it would break downstream users.

Reply:
Good afternoon :)
I added back the focal_length_ as deprecated and made it backwards compatible (it was used to specify HFOV, so it is the same as focal_length_x_).